ledger-api/grpc-definitions/com/digitalasset/ledger/api/v1/admin/party_management_service.proto

```diff
@@ -31,15 +31,22 @@ service PartyManagementService {
   // daml-on-sql: returns an identifier supplied on command line at launch time
   // daml-on-kv-ledger: as above
   // canton: returns globally unique identifier of the backing participant
-  rpc GetParticipantId (GetParticipantIdRequest) returns
-    (GetParticipantIdResponse);
+  rpc GetParticipantId (GetParticipantIdRequest) returns (GetParticipantIdResponse);
+
+  // Get the party details of the given party, or a ``NOT_FOUND`` error if the
+  // requested party is unknown.
+  rpc GetParty (GetPartyRequest) returns (GetPartyResponse);
```

Comment:
This is subsumed by `GetParties`. Do we really need 2 endpoints for this?

Reply:
Makes sense. I added the first one first and then realized it might be helpful to have both. For implicit party allocation, though it doesn't use the endpoints, it does need to request multiple party details at once.

Which endpoint does it make sense to keep?

Reply:
Spoke to @gerolf-da offline and decided to keep `GetParties`. I've removed `GetParty` and the whole query path associated with it.